<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsMatchingComponent</class>
 <widget class="QFrame" name="CSettingsMatchingComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_MatchingComponent">
   <item>
    <widget class="BlackGui::Editors::CMatchingForm" name="editor_MatchingForm">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_Butoons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_">
      <item>
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Reload">
        <property name="text">
         <string>reload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Editors::CMatchingForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/matchingform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_Reload</tabstop>
  <tabstop>pb_Save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
